e2e/callback_test.go

```diff
@@ -195,351 +196,322 @@ func (suite *CbTestSuite) SetupTest() {
-		suite.coordinator.RelayAndAckPendingPackets(suite.pathBC)
-		suite.coordinator.UpdateTime()
-	}
-}
-
-func sendIcsFromChainCToB(suite *CbTestSuite, nft, token_id, memo string, relay bool) {
-	msg := fmt.Sprintf(`{ "send_nft": {"cw721": "%s", "ics721": "%s", "token_id": "%s", "recipient":"%s", "channel_id":"%s", "memo":"%s"}}`, nft, suite.bridgeC.String(), token_id, suite.testerB.String(), suite.pathBC.EndpointB.ChannelID, memo)
-	_, err := suite.chainC.SendMsgs(&wasmtypes.MsgExecuteContract{
-		Sender:   suite.chainC.SenderAccount.GetAddress().String(),
-		Contract: suite.testerC.String(),
-		Msg:      []byte(msg),
-		Funds:    []sdk.Coin{},
-	})
-	require.NoError(suite.T(), err)
-
-	if relay {
-		suite.coordinator.UpdateTime()
-		suite.coordinator.RelayAndAckPendingPackets(suite.pathBC.Invert())
-		suite.coordinator.UpdateTime()
-	}
-}
-
-func sendIcsFromChainCToA(suite *CbTestSuite, nft, token_id, memo string, relay bool) {
-	msg := fmt.Sprintf(`{ "send_nft": {"cw721": "%s", "ics721": "%s", "token_id": "%s", "recipient":"%s", "channel_id":"%s", "memo":"%s"}}`, nft, suite.bridgeC.String(), token_id, suite.testerA.String(), suite.pathAC.EndpointB.ChannelID, memo)
-	_, err := suite.chainC.SendMsgs(&wasmtypes.MsgExecuteContract{
-		Sender:   suite.chainC.SenderAccount.GetAddress().String(),
-		Contract: suite.testerC.String(),
-		Msg:      []byte(msg),
-		Funds:    []sdk.Coin{},
-	})
-	require.NoError(suite.T(), err)
-
-	if relay {
-		suite.coordinator.UpdateTime()
-		suite.coordinator.RelayAndAckPendingPackets(suite.pathAC.Invert())
-	}
-}
-
-func queryTesterSent(t *testing.T, chain *wasmibctesting.TestChain, tester string) string {
-	resp := TesterResponse{}
-	testerSentQuery := TesterSentQuery{
-		GetSentCallback: EmptyData{},
-	}
-	err := chain.SmartQuery(tester, testerSentQuery, &resp)
-	require.NoError(t, err)
-	return *resp.Owner
-}
-
-func queryTesterReceived(t *testing.T, chain *wasmibctesting.TestChain, tester string) string {
-	resp := TesterResponse{}
-	testerReceivedQuery := TesterReceivedQuery{
-		GetReceivedCallback: EmptyData{},
-	}
-	err := chain.SmartQuery(tester, testerReceivedQuery, &resp)
-	require.NoError(t, err)
-	return *resp.Owner
-}
-
-func queryTesterNftContract(t *testing.T, chain *wasmibctesting.TestChain, tester string) string {
-	resp := ""
-	testerReceivedQuery := TesterNftContractQuery{
-		GetNftContract: EmptyData{},
-	}
-	err := chain.SmartQuery(tester, testerReceivedQuery, &resp)
-	require.NoError(t, err)
-	return resp
-}
-
-func queryTesterReceivedErr(t *testing.T, chain *wasmibctesting.TestChain, tester string) error {
-	resp := TesterResponse{}
-	testerReceivedQuery := TesterReceivedQuery{
-		GetReceivedCallback: EmptyData{},
-	}
-	err := chain.SmartQuery(tester, testerReceivedQuery, &resp)
-	return err
-}
-
-func (suite *CbTestSuite) TestSuccessfulTransfer() {
+func (suite *CallbackTestSuite) TestSuccessfulTransfer() {
 	memo := callbackMemo(nftSentCb(), "", nftReceivedCb(), "")
+	// A -> B token_id 2
```

Comment:
What's the difference between `sendIcsFromChainAToB` and `test_suite.Ics721TransferNft`? Seems to lookalike